nautobot/dcim/tests/test_graphql.py

```diff
@@ -83,3 +83,55 @@ def test_execute_query(self):
             self.assertEqual(
                 resp["data"]["devices"][0]["platform"]["network_driver_mappings"]["scrapli"], "cisco_iosxe"
             )
+
+        with self.subTest("device serial number query"):
+            non_empty_serial_device = Device.objects.first()
+            non_empty_serial_device.serial = "1234567890"
+            non_empty_serial_device.save()
+
+            # Test device serial query default behavior: serial__ie
+            query = 'query { devices (serial:" ") { name serial } }'
```

Comment:
that seems weird - we're specifying a space character as the serial value and expecting it to match devices with an empty serial? Do we or django-filters auto-strip whitespace or something?

Reply:
I think we are only appending non-empty values [here](https://github.com/nautobot/nautobot/blob/develop/nautobot/core/forms/fields.py#L280-L285). Honestly not sure if this is the correct way to approach this.